packages/core/src/browser/preferences/preference-provider.ts

```diff
@@ -9,7 +9,14 @@ import { Disposable, Event } from '../../common';
 
 export const PreferenceProvider = Symbol('PreferenceProvider');
 
+export enum Scope {
```

Comment:
It should be moved to preference-service.ts to be exposed and named `PreferenceScope`, `Scope` is too generic.

Reply:
done